specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -421,19 +418,6 @@
               "$ref": "#/definitions/InvoiceSectionListResult"
             }
           },
-          "202": {
```

Comment:
is this not a response coming from server? what's the reason for removing?

Reply:
correct. this was a bug. It is already fixed in master.